index.js

```diff
@@ -374,24 +376,40 @@ async function runTests(handle) {
 
     const testRunResponse = await SF.createTestRun(testParams);
     const testRunId = testRunResponse.data;
+    console.log(testRunId);
+
     let testRun = { status: "started" };
     const pollingDelay = 2000;
+    const spinner = new Spinner("%s Running tests..");
+    spinner.setSpinnerString("⠋⠙⠹⠸⠼⠴⠦⠧⠇⠏");
+    spinner.start();
 
     while (testRun.status === "started") {
-      await new Promise((resolve) => setTimeout(resolve, pollingDelay));
+      await new Promise((resolve) => {
+        setTimeout(resolve, pollingDelay);
+      });
       const response = await SF.fetchTestRun(testRunId);
       testRun = response.data;
     }
 
+    spinner.stop();
+    readline.clearLine(process.stdout);
+    readline.cursorTo(process.stdout, 0);
+
     if (testRun.status !== "completed") {
       console.error(testRun.error_message);
       process.exit(1);
     }
 
     if (testRun.result.length !== 0) {
```

Comment:
I wasn't able to test this. When it should happen ?

Reply:
Was still a snippet added by ENG before; but I think this should trigger if the test run fails on technical level (internal server error or a timeout error)